planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/scene.cpp

```diff
@@ -1587,7 +1540,10 @@ bool NormalLaneChange::getLaneChangePaths(
           }
         }
 
-        if (lane_changing_length + prepare_length > dist_to_end_of_current_lanes) {
+        if (
+          lane_changing_length + prepare_length +
+            common_data_ptr_->transient_data.next_lc_buffer.min >
```

Comment:
Just curious why did you add the next LC buffer here?

Reply:
I'm making an assumption that we want to do more than 1 lane changes. This is assuming that the end point of current and target lane align with each other. 

Additional note: some additional fix is needed if the end point is not aligned.

Reply:
but isn't this covered by the check [here](https://github.com/autowarefoundation/autoware.universe/pull/8849/files#diff-59cbf60b192ecddbbd9709016c0b4bd8846eda2ddbc91dc28937762391e13ff0R1548-R1550)